### ISSUE TITLE
Revise IX documentation

### ISSUE THREAD

**Issue by adam-a-a** [watertap-org/watertap#958]
### Description

Revise the [Solution Component Information](https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/ion_exchange_0D.html#solution-component-information) Section in ion exhange model's technical reference.

### Motivation

While reviewing an internal report on the IX model, I glanced at the documentation and realized that the code example given in the technical reference could benefit from a clearer example.

### Possible Implementation

I would specify that the IX model is currently only compatible with the MCAS property package (similar to what was done in GAC documentation), assuming that is true. Then, I would change the aforementioned code example to include instance creation of the MCAS property package and how the solute property data would be provided to the MCAS property package, before building the instance of the IX model.

### Additional Context

Basically, imagine that a brand-new user went straight to the IX documentation. Would the provided example be immediately helpful?

**Comment by adam-a-a** [comment]
Revisiting the aforementioned documentation, and it appears to have been resolved
